autoscaler/cluster.py

```diff
@@ -917,6 +922,9 @@ def get_pods_to_schedule(self, pods):
         # unassigned and feasible
         pods_to_schedule = {}
         for pod in pending_unassigned_pods:
+            age = (datetime.datetime.now(pytz.utc) - pod.creation_time).total_seconds()
```

Comment:
should we just reuse the same datetime object for all pods instead of creating a new one every iteration?